packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -647,10 +681,25 @@ class PageView extends StatefulWidget {
     this.onPageChanged,
     @required this.childrenDelegate,
     this.dragStartBehavior = DragStartBehavior.start,
+    this.allowImplicitScrolling = false,
   }) : assert(childrenDelegate != null),
+       assert(allowImplicitScrolling != null),
        controller = controller ?? _defaultPageController,
        super(key: key);
 
+  /// Controls whether the widget's pages will respond to
+  /// [RenderObject.showOnScreen], which will allow for implicit accessibility
+  /// scrolling.
+  ///
+  /// With this flag set to false, when accessibility focus reaches the end of
+  /// the current page and a user performs another implicit scroll action,
```

Comment:
the "implicit scroll action" is moving a11y focus to the next element. Maybe we should say:

With this flag set to false, when accessibility focus reaches the end of current page and the user attempts to move it to the next element, the focus will traverse to the next widget outside of the page view.

With this flag set to true, when accessibility focus reaches the end of the current page and user attempts to move it to the next element, focus will traverse to the next page in the page view.

Reply:
I like that better, thanks - done.